### PR TITLE
Luotu kuorikomentosarja helpottamaan tunkkausta

### PR DESCRIPTION
**Pull request by teemupaloniemi**
Yhdistetään tunkkaaminen yhteen paikkaan, jolloin kitketään mahdollisia kehitysympäristön poikkeamien aiheuttamia ongelmia pois. TODO: Muut käyttöjärjestelmät.

### CODE REVIEW

**Review by 45spoons — APPROVED** [review]
This is a great feature that I hope will help onboarding new contributors. At this stage I believe this works. What I would like to see improved in the future for tis feature is better support for "unsupported" operating systems.
To give an example:
Running this on Arch, the tool could tell me what dependencies I still need to install, instead of stating my OS is unsupported.

File: readme.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ To compile and open locally:
 3. Initialize git submodules `git submodule init && git submodule update`
 4. Run `hugo server` and you should be good to go 🎉
 
+Optionally run `startup.sh`.
```
Comment (lurppa) [inline readme.md:14]: How about something like
```markdown
Or optionally, on some supported Unix based environments _(currently MacOS, Debian and Fedora)_, you can just run `startup.sh`.
```
to be more percise?

**Review by lurppa — COMMENTED** [review]
A welcome addition for sure :+1:

File: startup.sh
Comment (teemupaloniemi) [inline startup.sh:1]: For some reason hugo was not installing if it was coupled with update. This is a partial fix and assumes that the system is up to date.

Comment (teemupaloniemi) [inline startup.sh:1]: The `hugo server --config="hugo.toml"` crashed if the lfs content was not pulled. This did not probably come up in the original setup if the user installed git lfs before cloning the repo.

**Pull request merged**